libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/internal/MapboxOnboardRouter.kt

```diff
@@ -50,37 +52,22 @@ class MapboxOnboardRouter(
 ) : Router {
 
     private companion object {
-        private const val TAG = "MapboxOnboardRouter"
+        private val loggerTag = Tag(MapboxOnboardRouter::class.java.simpleName)
```

Comment:
We can even hardcode it ¯\\\_(ツ)\_/¯

```suggestion
        private val loggerTag = Tag("MapboxOnboardRouter")
```

Reply:
SGTM